TheJumpCloud/support
# CUT-4281: Set-JCUser Null Manager fix

## Issues
* [CUT-4281](https://jumpcloud.atlassian.net/browse/CUT-4281) - Set-JCUser Null Manager fix

## What does this solve?
Due to the way that PowerShell treats string datatypes, you are unable to truly set a string parameter to null. The manager field requires that the manager field be set to strictly null, an empty string will not work.

In order to solve for this, we simply need to remove the string datatype on the manager parameter. This will default that parameter to accepting 'object' input, which will work in this scenario

## Is there anything particularly tricky?
N/A

## How should this be tested?
1. Import the module
2. Attempt to set a user's manager to $null
```powershell
Set-JCUser -id <UserID> -manager $null
```
4. This should now remove any existing values in the user's manager field


[CUT-4281]: https://jumpcloud.atlassian.net/browse/CUT-4281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
